src/cfnlint/data/ExtendedSpecs/all/03_value_types/aws_cassandra.json

```diff
@@ -0,0 +1,18 @@
+[
+ {
+  "op": "add",
+  "path": "/ValueTypes/AWS::Cassandra::Keyspace.KeyspaceName",
+  "value": {
+   "StringMax": 255,
+   "StringMin": 3
+  }
+ },
+ {
+  "op": "add",
+  "path": "/ValueTypes/AWS::Cassandra::Table.TableName",
```

Comment:
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-cassandra-table.html#cfn-cassandra-table-tablename

Same we have the pattern that defines the length and the pattern and length descriptions don't match.